identity/app/controllers/HealthCheck.scala

```diff
@@ -4,5 +4,5 @@ import conf.{AllGoodCachedHealthCheck, NeverExpiresSingleHealthCheck}
 import play.api.libs.ws.WSClient
 
 class HealthCheck(wsClient: WSClient) extends AllGoodCachedHealthCheck(
-  NeverExpiresSingleHealthCheck("/signin")
+  NeverExpiresSingleHealthCheck(routes.ReauthenticationController.renderForm(None).url)
```

Comment:
I don't know much about this endpoint. As long as it is accessible without being authenticated I guess it would do